Sources/SpeziScheduler/Scheduler.swift

```diff
@@ -39,6 +39,7 @@ public class Scheduler<Context: Codable>: Module, EnvironmentAccessible, Default
         taskList.tasks
     }
 
+    public var notificationContentClosure: ((Task<Context>, Event) -> UNNotificationContent)?
```

Comment:
Wondering if we should make this public and modifiable after the initial configuration. If we make it publicly modifiable we would need to re-pass this back to all of the current existing tasks in a `didSet` clause.